components/fxa-client/src/oauth.rs

```diff
@@ -92,6 +92,18 @@ impl FirefoxAccount {
     ///
     /// * `scopes` - Space-separated list of requested scopes.
     pub fn begin_oauth_flow(&mut self, scopes: &[&str]) -> Result<String> {
+        return self.begin_oauth_flow_with_context(scopes, "oauth");
```

Comment:
Can we externalize `"oauth"` to a constant at the beginning of that file with a comment?